docs/bridge/messages.md

```diff
@@ -0,0 +1,129 @@
+# Introduction
+This document outlines how the messaging works between starlight and ethereum. It will describe each type of message supported by either side and then 
+will describe the general flow of the messages from creation to handling of it.
+
+# Commits of both tanssi-symbiotic and dancelight runtime referenced
+- Tanssi-symbiotic: https://github.com/moondance-labs/tanssi-symbiotic/commit/1428c2a436a54edb154274bebc94daf390ecf125
+- Tanssi: https://github.com/moondance-labs/tanssi/commit/ac9c25b0e3f2a45a7def57b556b8b666d7949a3f
+
+# ReceiveValidators
+Sends account ids of operators from symbiotic to starlight.
+
+## Direction
+Ethereum -> Starlight
+
+## Payload
+```solidity
+return bytes.concat(
+    bytes4(0x70150038), // Magic bytes
+    bytes1(uint8(Message.V0)), // Message versioning bytes
+    bytes1(uint8(OutboundCommandV1.ReceiveValidators)), // Enum byte
+    ScaleCodec.encodeCompactU32(operatorsCount), // Operator count
+    operatorsFlattened, // Operators array
+    ScaleCodec.encodeU64(uint64(epoch)) // timestamp
+);
```

Comment:
Add link to github where this is defined, so when this document becomes outdated we can still find the new definition. Same for other code snippets

Reply:
Sure